### ISSUE TITLE
ERR_AUTH from start-rpc

### ISSUE THREAD

**Issue by hazinstore** [blockchain/service-my-wallet-v3#171]
Node v7.5.0
npm 4.1.2

Using...
>blockchain-wallet-service start-rpc -p 8332 in one terminal and

>curl --user 'REDACTED-e434-4208-bf27-REDACTED' --data-binary '{"jsonrpc": "1.0", "id":"curltest", "method": "getinfo", "params": [] }' \
-H 'content-type: text/plain;' http://127.0.0.1:8332/

In the second terminal...
It prompts me for my password, which I enter correctly then I get the following output...
>{"jsonrpc":"2.0","error":{"code":-32603,"message":"ERR_AUTH"},"id":"curltest"}

This happens every time and every time I also get an email about it...
>An attempt to login to your blockchain.info wallet was made from an unknown browser. Please confirm the following details are correct:
Time: 2017-02-04 11:38:14
IP Address: REDACTED
Browser: Unknown
User Agent: node-fetch/1.0 (+https://github.com/bitinn/node-fetch) 
Please check the ip address and browser carefully. If the above details are correct click the following link to approve the request:
https://blockchain.info/wallet/#/authorize-approve/REDACTED
If this login attempt was not made by you it means someone visited your wallet login page from an unrecognised browser. It may be an indication you have been the target of a phishing attempt and might want to consider moving your funds to a new wallet.

Any help would be greatly appreciated.  Thanks!

The curl command is taken from the official JSON RPC docs for bitcoin client.

**Comment by jtormey** [comment]
If you have your email verified in the wallet, then we'll require that the authorization link is clicked before a login can happen from a new location (the wallet service in this case). Although, the service does not store the new session token after approval, which is probably why you get an authorization email every time, and login is still denied.

To get around this, you can add your IP address to the "IP whitelist" setting in the wallet. If the requesting IP is in the whitelist, we don't send the auth email.

This brings up an interesting point, it might be a good idea to temporarily store the session token while the service is running. That way, only one auth email is required and you don't have to change the IP whitelist. Will think on it...

**Comment by metroyanno** [comment]
@jtormey, and I have this problem

**Comment by Lalitpatadiya12** [comment]
Why error comes when I am sending bitcoin face the error : "Wallets that require email authorization are currently not supported in the Wallet API. Please disable this in your wallet settings, or add the IP address of this server to your wallet IP whitelist" 

Is their any way to bypass this error.

**Comment by jtormey** [comment]
Yes, see my original reply above. You can either remove your email from the wallet (in preferences), or add your server IP to the IP whitelist (in security settings).

**Comment by igor15** [comment]
ты кто мудень17.03.2017, 20:29, "Justin Tormey" <notifications@github.com>:Yes, see my original reply above. You can either remove your email from the wallet (in preferences), or add your server IP to the IP whitelist (in security settings).

**Comment by kitsunde** [comment]
>  You can either remove your email from the wallet

@jtormey There's no way of removing the email in preferences.  I can press change, and I can edit it, but not remove it.

**Comment by jtormey** [comment]
By "remove email" I meant "change so that it becomes unverified". Sorry for the confusion.

**Comment by proBeejay141** [comment]
Am trying to use blockchain-wallet-service with bitcoinLib,  it alway return 0, whenever i called getBalance method

**Comment by fercreek** [comment]
I added my IP but it doesn't work. Does someone find how to remove email?
